server/src/rooms/schema/MyRoomState.ts

```diff
@@ -26,7 +27,7 @@ export class PlayerState extends Schema {
   @type("number") health: number = 100;
   @type("uint8") healthState: number = PlayerHealthState.ALIVE;
 
-  @type("uint32") coins: number = 0;
+  @type("uint32") coins: number = 30;
```

Comment:
This was just for testing forgot to take out
but maybe  a bit of starting money actually isn't a bad idea